core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -391,6 +392,7 @@ object KafkaConfig {
   val AdvertisedListenersProp = "advertised.listeners"
   val ListenerSecurityProtocolMapProp = "listener.security.protocol.map"
   val ControlPlaneListenerNameProp = "control.plane.listener.name"
+  val SocketNagleDisableProp = "socket.nagle.disable"
```

Comment:
I think I would call this `socket.tcp.no.delay`.

Reply:
I take `socket.nagle.disable` from librdkafka project. But ok, I rename it to `socket.tcp.no.delay`. Perhaps this is clearer.